### ISSUE TITLE
Adding multiple activities to a route

### ISSUE THREAD

**Issue by Weblors** [derickbailey/mustbe#10]
Hi,

I want to add an array of activities on a route instead of just one activity.

**Comment by derickbailey** [comment]
Hi @Weblors I'm not sure this is a good idea. Having multiple names for the same activity doesn't make it a new activity. It just means you have multiple names for it.

Judging from nothing more than the request, it looks like you're trying to replace the idea of an activity with a role - to say that all of these roles can execute this route. This is exactly the opposite of what you wan to do, and why I built mustbe. If you haven't read it, yet, I'd suggest going back to [my article on activity based checks vs role based checks](https://lostechies.com/derickbailey/2011/05/24/dont-do-role-based-authorization-checks-do-activity-based-checks/) to get a better understanding of where mustbe is coming from.

To sum it up, I think having multiple names for the same activity is a bad idea. You should have a single name for the activity that is represented by this route.

...

Ok, now that I've made all of those assumptions :D 

I clearly don't know exactly what you're attempting to do, or why. If you can clarify why you want an array of activity names for one route, I'd love to hear why. Perhaps there is a valid use case for this.

**Comment by Weblors** [comment]
Hi @derickbailey Thanks for the clarification. I've read the article and some other literature and I must say I find the idea of using activity-based authorization quite interesting. I've already architected my app to adapt to activity-based authorization and I've been finding it a lot better doing it this way.

Thanks for all your help :D
